libs/stokesian_dynamics/sd.hpp

```diff
@@ -1302,12 +1311,20 @@ struct thermalizer {
     The mean square displacement during a time step of length \f$\Delta t\f$,
     and along one degree of freedom is given by
 
-            \f[\langle x^2 \rangle / \Delta t = 2 D\f]
-
-    That way we can determine the distribution of the random displacement
-    that our system experiences along one of its many degrees of freedom. In
-    our case, \f$\mu\f$ is a matrix and we need its square root. The latter is
-    obtained via Cholesky decomposition.
+            \f[\langle \Delta x^2 \rangle / \Delta t = 2 D\f]
+    
+    By combining these two equations we get the random displacement, which we
+    can then directly use to propagate the particles:
+    
+            \f[\Delta x = \sqrt(2 k_B T \Delta t) \mu^{1/2} \cdot \Phi]
+
+    where \f[\Phi] is a random number drawn from a zero mean and unit variance
```

Comment:
- in a few places in this docstring, the LaTeX math closing markup `]` should actually read `\f]`
- if I remember correctly, `\f[...\f]` is used for formula that appear centered outside a paragraph (like in the expression of `\f[ \Delta x = ...` at L1314), while `\f$...\f$` are used for small inline formula appearing in the running text (e.g. `\f$ \Phi \f$` inside the paragraph at L1321)

Reply:
yep I have already fixed that, your earlier modifications made clear how it is done, only I didn't get it the first time